azure/packages/external-controller/src/app.ts

```diff
@@ -4,12 +4,14 @@
  */
 import { IFluidContainer, IValueChanged, SharedMap } from "fluid-framework";
 
+import { DevtoolsLogger, initializeDevtools } from "@fluid-experimental/devtools";
 import {
 	AzureClient,
 	AzureContainerServices,
 	AzureLocalConnectionConfig,
 	AzureRemoteConnectionConfig,
 } from "@fluidframework/azure-client";
+import { TelemetryNullLogger } from "@fluidframework/telemetry-utils";
```

Comment:
Added to demonstrate providing a logger to the client, and how to wrap an existing logger in the devtools logger.